sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -72,6 +79,22 @@ export type BeginTrainingOptions<T> = TrainingFileFilter & {
 // @public
 export type ContentPollerLike = PollerLike<PollOperationState<RecognizeContentResultResponse>, RecognizeContentResultResponse>;
 
+// @public
+export type CopyAuthorization = {
+    resourceId: string;
+    resourceRegion: string;
+} & CopyAuthorizationResultModel;
+
+// @public
+export interface CopyAuthorizationResultModel {
+    accessToken: string;
+    expirationDateTimeTicks: number;
```

Comment:
I feel like this shouldn't be exposed to users. I barely know what a tick is or how to handle it. Can we lift this to `expiresOn` for the user?

Reply:
There's `expiresOn` but commented out for now as currently it will not work in production (64 bit interger from the service).  It's being changed to seconds and 32-bit when GA.  We can expose it then.

Reply:
Ok that sounds good.